packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -436,13 +432,9 @@ const PostsPagePostHeader = ({post, answers = [], dialogueResponses = [], showEm
       <div className={classes.tagSection}>
         {!post.shortform && !post.isEvent && !hideTags && 
         <AnalyticsContext pageSectionContext="tagHeader">
-          <FooterTagList post={post} hideScore allowTruncate overrideMargins={true}/>
+          <FooterTagList post={post} hideScore allowTruncate overrideMargins={true} annualReviewMarketInfo={annualReviewMarketInfo} />
```

Comment:
My current guess is that this change should be reverted, though I don't know for sure why it was implemented here (it's in the regular post header, not the splash page post header, so it doesn't seem related to this PR).

Reply:
(@rjmk)